drv/lpc55-swd/src/main.rs

```diff
@@ -913,7 +913,7 @@ impl ServerImpl {
         // for the correct count.
         //
         // Round up here just to be safe
-        let rounded = ((cnt + 7) / 8) * 8;
+        let rounded = cnt.div_ceil(8) * 8;
```

Comment:
I _think_ this is correct but spot check this to make sure we have done bytes to bits correctly?

Reply:
It's equivalent in the main but if `cnt` is greater than `usize::MAX - 7` the prior code panicks. `div_ceil` avoids that case by only adding to a value that's already been divided.

Reply:
With the move to the newer compiler imminent, I'd take a look at `.next_multiple_of`, myself.